tests/src/rule-tester.js

```diff
@@ -0,0 +1,62 @@
+import { RuleTester } from 'eslint';
+import { version as eslintVersion } from 'eslint/package.json';
+import semver from 'semver';
+
+export const usingFlatConfig = semver.major(eslintVersion) >= 9;
+
+export function withoutAutofixOutput(test) {
+  return { ...test, output: usingFlatConfig ? null : test.code };
```

Comment:
```suggestion
  return { ...test, ...(usingFlatConfig || { output: test.code }) };
```

Reply:
huh good point - for some reason I forgot that you can spread booleans 😅

Reply:
ah right so that isn't actually equivalent because it results in no object when we need to be explicitly setting `output` to either `null` or `test.code`; with this change, we spread an empty object meaning that `output` never changes